net-p2p/retroshare/retroshare-0.6.7.ebuild

```
@@ -0,0 +1,133 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic desktop qmake-utils xdg
+
+DESCRIPTION="Friend to Friend secure communication and sharing application"
+HOMEPAGE="https://retroshare.cc"
+SRC_URI="http://download.opensuse.org/repositories/network:/retroshare/Debian_Testing/retroshare-common_${PV}.orig.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/RetroShare"
+# NOTE: GitHub releases/archive is impractical to build so we use the OBS repo
+
+LICENSE="AGPL-3 Apache-2.0 CC-BY-SA-4.0 GPL-2 GPL-3 LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="keyring cli +gui +jsonapi libupnp +miniupnp +service +sqlcipher plugins"
+
+REQUIRED_USE="
+	|| ( gui service )
+	?? ( libupnp miniupnp )
+	service? ( || ( cli jsonapi ) )"
```

```suggestion
REQUIRED_USE="
	|| ( gui service )
	?? ( libupnp miniupnp )
	plugins? ( gui )
	service? ( || ( cli jsonapi ) )"
```

I noticed the build system stops if `+plugins` is enabled while `+gui` is not.